### PR TITLE
(PC-22466) feat(video): add description and publication date

### PR DESCRIPTION
**Pull request by tanguyprache**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-22466

## Checklist

I have:

- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

## Screenshots

**delete** *if no UI change*

| Type         | Screenshot |
| :--------------- | :----: |
| With description              |Simulator Screen Shot - iPhone 13 - 2023-06-22 at 18 25 56| 
| Without description              |Simulator Screen Shot - iPhone 13 - 2023-06-22 at 18 31 43| 

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/home/components/modals/VideoModal.tsx [hunk 1]
```diff
@@ -49,9 +45,15 @@ export const VideoModal: React.FC<VideoModalProps> = (props) => {
         <Spacer.Column numberOfSpaces={2} />
         <Typo.Title3>{props.title}</Typo.Title3>
         <Spacer.Column numberOfSpaces={2} />
-        <StyledCaptionDate>{`Publiée le ${publicationDate}`}</StyledCaptionDate>
-        <Spacer.Column numberOfSpaces={2} />
-        <StyledBody>{descriptionText}</StyledBody>
+        <StyledCaptionDate>{`Publiée le ${formatToFrenchDate(
+          props.videoPublicationDate
+        )}`}</StyledCaptionDate>
+        {props.videoDescription ? (
```
Comment (pierrecanthelou) [inline src/features/home/components/modals/VideoModal.tsx:51]: Pourquoi pas **!!props.videoDescription && ...** ?

**Review by pierrecanthelou — COMMENTED** [review]
Une question syntaxique :)

File: src/features/home/components/modals/VideoModal.tsx (hunk 1 above)
Comment (tanguyprache) [reply src/features/home/components/modals/VideoModal.tsx]: Oui vieille habitude de n'utiliser que du ternary pour le conditionnal rendering :)

**Review by S4andro — APPROVED** [review]

File: src/features/home/components/modals/VideoModal.tsx (hunk 1 above)
Comment (pierrecanthelou) [reply src/features/home/components/modals/VideoModal.tsx]: Ok je te laisse voir avec @anoukhello et/ou les autres devs ce qui est "recommandé par passCulture" :)

**Pull request merged**